finalReport.csv

```diff
@@ -0,0 +1 @@
+fruit,quantitybanana, 0apple, 0
```

Comment:
The report should be like, let's fix style
    fruit,quantity
    banana,152
    apple,90

Reply:
My report looks like that:
fruit,quantity
banana,152
apple,90